openshift_sti_images/overview.adoc

```diff
@@ -0,0 +1,4 @@
+= Overview
+{product-author}
+{product-version}
+:data-uri:
```

Comment:
Can you please throw in a paragraph, like:

This topic group includes information on the different Source-to-Image tools available for OpenShift users.